batchflow/models/torch/base.py

```diff
@@ -1070,14 +1078,20 @@ def _train(self, *args, fetches=None, train_mode='', accumulate_grads=True, sync
                 else:
                     self.sync_counter += 1
 
-                if decay:
-                    if step.get('current_iter', 0) >= step['n_iters']:
-                        decay.step()
-                        step['current_iter'] = 0
-                    step['current_iter'] = step.get('current_iter', 0) + 1
+                curr_lr = [group['lr'] for group in optimizer.param_groups]
+                self.iter_info.setdefault('lr', []).append(curr_lr)
+
+                if decays:
+                    for decay, step_meta in zip(decays, decay_step):
```

Comment:
Why not `decay_params` or somewhat like that instead of weird `*_meta` variable names?

Reply:
done